src/MetricsUploader/Result.cpp

```diff
@@ -27,6 +27,10 @@ std::string GetErrorMessage(Result result) {
       return "Connection to metrics uploader wasn't closed";
     case kNoError:
       return "No errors";
+    case kUnknownStatusCode:
+      return "Unknown status code";
```

Comment:
I'm not sure that it's very clear what was the error here. Maybe we should add a bit more details here (specify that the OrbitLogEvent::StatusCode in library doesn't have the code)?